migrator.sh

```diff
@@ -293,10 +301,38 @@ pull_images_from_v1() {
   for i in ${FULL_IMAGE_LIST}
   do
     push_pull_image "pull" "${V1_REGISTRY}/${i}"
+    retag_image "${V1_REGISTRY}/${i}" "${V2_REGISTRY}/${i}"
+    push_pull_image "push" "${V2_REGISTRY}/${i}"
```

Comment:
Unfortunately this will not work in all cases.  Specifically where you do not want to retag the images because they are using the same hostname for a v1 and v2 registry.